pyedr/pyedr/pyedr.py

```diff
@@ -464,7 +469,30 @@ def read_edr(path: str, verbose: bool = False) -> read_edr_return_type:
     return all_energies, all_names, times
 
 
-def edr_to_dict(path: str, verbose: bool = False) -> Dict[str, np.ndarray]:
+def get_unit_dictionary(path: str) -> Dict[str, str]:
+    """Creates an EDRFile object which executes the :func:`do_enxnms`
+    method. This reads the names and units of the EDR data, which is returned
+    as a dictionary mapping column names (str) to unit names (str).
+
+    Parameters
+    ----------
+    path : str
+        path to EDR file to be read
+
+    Returns
+    -------
+    unit_dict: Dict[str, str]
+        A dictionary mapping the term names to their units.
+    """
+    edr_file = EDRFile(str(path))
+    unit_dict = {'Time': "ps"}
+    for nm in edr_file.nms:
+        unit_dict[nm.name] = nm.unit
+    return unit_dict
+
+
+def edr_to_dict(path: str, verbose: bool = False) -> (Dict[str, np.ndarray],
```

Comment:
I'm not sure I understand what's going on with this PR, didn't you say that you wanted to not return a tuple here, and therefore added `get_unit_dictionary`, but this is still happening?

Reply:
I really need to stop changing code without changing the documentation to match it..... thanks!